src/tree/SubscriptionTreeItem.ts

```diff
@@ -76,6 +81,26 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         return result.data;
     }
 
+    public async pickResourceGroup(context: IActionContext, options: PickResourceGroupOptions & { canPickMany: true }): Promise<ResourceGroupTreeItem[]>;
+    public async pickResourceGroup(context: IActionContext, options: PickResourceGroupOptions & { canPickMany: false }): Promise<ResourceGroupTreeItem>;
+    public async pickResourceGroup(context: IActionContext, options: PickResourceGroupOptions): Promise<ResourceGroupTreeItem | ResourceGroupTreeItem[]> {
+        await this.getCachedChildren(context);
+        const rgsTreeMap = this.cache.getTreeMap(context, 'resourceGroup');
+        const tis = (await context.ui.showQuickPick((Object.values(rgsTreeMap).filter((groupTreeItem) => groupTreeItem instanceof ResourceGroupTreeItem) as ResourceGroupTreeItem[]).map((rg: ResourceGroupTreeItem): IAzureQuickPickItem<ResourceGroupTreeItem> => ({
```

Comment:
Might be worth to include this implementation in the azext-utils package.  I know in the interest of time, might not be feasible now, but I could see this being useful in other commands.